decentraland/unity-renderer
# Fix: facial feature refresh

## What does this PR change?

Fix #2342 
Hides facial features when needed

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=fix/facial-feat-hide
2. Verify other characters facial feature visibility
3. Try to equip something that should hide facial feats

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
